include/aos/sm/networkmanager.hpp

```diff
@@ -513,11 +538,40 @@ class NetworkInterfaceManagerItf {
     /**
      * Gets route list.
      *
-     * @param ifname interface name.
      * @param[out] routes routes.
      * @return Error.
      */
-    virtual Error GetRouteList(const String& ifname, Array<RouteInfo>& routes) const = 0;
+    virtual Error GetRouteList(Array<RouteInfo>& routes) const = 0;
+};
+
+/**
+ * Network interface factory interface.
+ */
+class NetworkInterfaceFactoryItf {
```

Comment:
Let's rename to `InterfaceFactoryItf` it is already in networkmanager namespace.
Also rename `NetworkInterfaceManagerItf` to `InterfaceManagerItf` for same reason.

Reply:
Done